src/util/childProcess.js

```diff
@@ -48,21 +74,24 @@ module.exports = class ChildProcess {
     }
   }
 
+  isTextWhiteListed(text) {
```

Comment:
return STDOUT_WHITE_LIST.some(item => text.includes(item))
```suggestion
  isTextWhiteListed(text) {
    return STDOUT_WHITE_LIST.some(item => text.includes(item))
  }
}
```
?